packages/server/src/__tests__/plugin/subscriptionCallback/index.test.ts

```diff
@@ -1082,9 +1300,16 @@ async function startSubscriptionServer(
   const pubsub = new PubSub();
   const server = new ApolloServer({
     plugins: [
-      ApolloServerPluginSubscriptionCallback(
-        opts?.logger ? { logger: opts.logger } : undefined,
-      ),
+      ApolloServerPluginSubscriptionCallback({
+        // set some reasonable testing defaults
+        retry: {
+          retries: 5,
+          factor: 1,
+          maxTimeout: 100,
```

Comment:
If we go with exponential backoff, maxTimeout should be `1000` maybe?

Reply:
Yeah, this is just for testing so I went with a simple config, but we still should provide some defaults we consider reasonable and document them.